modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -855,12 +857,8 @@ private void _testCreateManyToManyObjectRelationshipTable(
 			objectRelationship.getDBTableName(),
 			reverseObjectRelationship.getDBTableName());
 
-		Assert.assertEquals(
-			StringBundler.concat(
-				"R_", objectRelationship.getCompanyId(),
-				objectDefinition.getShortName(), "_",
-				relatedObjectDefinition.getShortName(), "_", name),
-			objectRelationship.getDBTableName());
+		Assert.assertTrue(
+			_matchesWithRegex(objectRelationship.getDBTableName()));
```

Comment:
@thallescezar can we inline here as the method `_matchesWithRegex` is only used once? i.e.:

```
Matcher matcher = _manyToManyObjectRelationshipTableNamePattern.matcher(objectRelationship.getDBTableName());

Assert.assertTrue(matcher.matches());
```